tools/pytest_parallel

```diff
@@ -76,6 +77,10 @@ while [ "$#" -gt 0 ]; do
             python="$2"
             shift; shift
             ;;
+        --tsan )
+            saved_tsan_options=$TSAN_OPTIONS
```

Comment:
Can we check for `TESTUTIL_TSAN` instead of introducing a new flag?

Reply:
I will need to get the current tsan flag options so that I can add `log_path=$(git rev-parse --show-toplevel)/tsan_logs_$pyfile:exitcode=0`. Therefore I used the tsan flag. Let me know if you don't like this i will also use `TESTUTIL_TSAN `